### DISCUSSION TITLE
GFN2-xTB geometry optimization doesn't converge

### DISCUSSION THREAD

**Discussion by Teo-98** [grimme-lab/xtb#814] [General]
Good morning, I'm actually using xTB and I'm enjoying it but I found a problem with a calculation: I tried to optimize a short RNA strand (AGGC) with a Fe(phen)3 complex intercalated between G's but the software says to me the error:
"[ERROR] Program stopped due to fatal error
-7- Geometry optimization failed
-6- xtb_geoopt: Trying to recover from failed geometry optimization
-5- xtb_geoopt: Geometry optimization did not converge
-4- optimizer_ancopt: Could not relax structure
-3- optimizer_relax: SCF not converged, aborting...
-2- xtb_calculator_singlepoint: Electronic structure method terminated
-1- scf: Self consistent charge iterator did not converge"
I tried to increment the SCC steps but it didn't work and I also tried to change the start geometry. I actually work with windows and the version is 6.5.1. I attach the xyz file in txt format to show the initial structure
Thanks for your help,
[AGGC_Fe.txt](https://github.com/grimme-lab/xtb/files/11516190/AGGC_Fe.txt)

Kind regards.

**Comment by haneug** [comment]
I was able to converge a GFN1-xTB optimization with the following:
1. Preoptimization with the GFN Force field (GFN-FF): xtb struc.xyz --gfnff --opt
2. Optimization with GFN1-xTB and alpb for water (to screen the charges): xtb xtbopt.xyz --gfn 1 --alpb water --opt
Here is the result:
[xtbopt.txt](https://github.com/grimme-lab/xtb/files/11516279/xtbopt.txt)

Best
Hagen

**Comment by Teo-98** [comment]
Thanks, mr. Hagen. I appreciated a lot your kindness and speed in answering me. 
I'm actually doing this for my master's degree thesis. It's outstanding! 
Best 
Teo
